notebooks/05_Kubeflow_Pipeline/src/Pipeline_mnist/src/katib_launch_args.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+def argugments_provide():
+    objectiveConfig = {
+      "type": "minimize",
+      "goal": 0.001,
+      "objectiveMetricName": "loss",
+    }
+    
+    algorithmConfig = {"algorithmName" : "random"}
+
+    parameters = [
+      {"name": "--tf-learning-rate", "parameterType": "double", "feasibleSpace": {"min": "0.01","max": "0.03"}},
+      {"name": "--tf-batch-size", "parameterType": "discrete", "feasibleSpace": {"list": ["16", "32", "64"]}},
+    ]
+    
+    rawTemplate = {
+      "apiVersion": "kubeflow.org/v1",
+      "kind": "TFJob",
+      "metadata": {
+         "name": "{{.Trial}}",
+         "namespace": "{{.NameSpace}}"
+      },
+      "spec": {
+        "tfReplicaSpecs": {
+          "Chief": {
+            "replicas": 1,
+            "restartPolicy": "OnFailure",
+            "template": {
+              "spec": {
+                "containers": [
+                {
+                  "command": [
+                    "sh",
+                    "-c"
+                  ],
+                  "args": [
+                    "python /opt/model.py --tf-train-steps=200 {{- with .HyperParameters}} {{- range .}} {{.Name}}={{.Value}} {{- end}} {{- end}}"
```

Comment:
This is simpler but the problem is we encapsulate too many details. The user running the example doesn't even know what's inside the container? My point is do we have something available and we can only fill in 

- command 
- argument
- image
- PS/Wroker replicas 
- PVC

This works for me as well. My concern is user finish running that, they won't have any get knowledge of what they ran.

Reply:
From my side, I was considering the tradeoff between simplicity and knowledge for users. But I fix this in the `katib_launch_args.py`.